lib/rex/post/sql/ui/console/interactive_sql_client.rb

```diff
@@ -100,10 +107,15 @@ def _multiline
           next true
         end
 
-        # In the case only a stop word was input, exit out of the REPL shell
-        finished = (multiline_input.split.count == 1 && stop_words.include?(multiline_input.split.last))
+        if multiline_input.split.count == 1
+          # In the case only a stop word was input, exit out of the REPL shell
+          finished = stop_words.include?(multiline_input.split.last)
+          # In the case when only a help word was input call the help command
+          help = help_words.include?(multiline_input.split.last)
+        end
 
         finished || multiline_input.split.last&.end_with?(';')
+        help || multiline_input.split.last&.end_with?(';')
```

Comment:
```suggestion
        finished || help || multiline_input.split.last&.end_with?(';')
```

Or something else maybe